styleguide/css/main.scss

```diff
@@ -1,31 +1,72 @@
-body {
-  margin-top: 60px;
+$white: #fff;
+$light-gray-white: #efefef;
+$dark-gray: #333;
+$medium-gray: #666;
+
+
+.mainContainer {
+  padding: 40px 0;
+  transition: margin-left .5s;
```

Comment:
Its the transition of sliding the main container over to expose the navigation. Is there a better way? Should we discuss this in our meeting for the doc review?